src/shared/dfu/index.ts

```diff
@@ -635,9 +635,12 @@ export class WebDFU {
       (ignoreErrors || dfuStatus.state !== dfuCommands.dfuERROR)
     ) {
       // eslint-disable-next-line no-await-in-loop
-      await asyncSleep(dfuStatus.pollTimeout);
-      // eslint-disable-next-line no-await-in-loop
-      dfuStatus = await this.getStatus();
+      await asyncSleep(1);
```

Comment:
Is this arbitrary?

Reply:
The goal is to avoid clogging the worker with real busy wait.

Reply:
Ah! If that's the reasoning, I think `asyncSleep(0)` also works, as `setTimeout` always runs on the next event loop execution

Reply:
Nice, thx! I'll try it out then!

Reply:
Done, seems to work properly, thx!